environment.yml

```diff
@@ -0,0 +1,10 @@
+name: pangeo
+dependencies:
+- distributed
+- dask
+- dask-jobqueue
+- nodejs
+- jupyterlab
+- jupyter-server-proxy
+- pip:
+  - dask_labextension
```

Comment:
`dask-labextension` is now in `conda-forge` channel in case you are interested in avoid pip.